src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -139,6 +139,65 @@ struct ConsentsUpdater
 };
 
 /**
+ * @brief Checks current application id total results in policy check results
+ */
+struct AppPolicyResultsChecker {
+  AppPolicyResultsChecker(
+      policy_table::ApplicationPolicies::const_iterator app_params)
+      : app_params_(app_params)
+      , is_notify_system_(false)
+      , is_send_permissions_(false)
+      , is_consent_needed_(false) {}
+
+  void operator()(const policy::CheckAppPolicyResults::value_type& value) {
+    if (value.first != app_params_->first) {
+      return;
+    }
+
+    using namespace policy;
+    switch (value.second) {
+      case RESULT_NO_CHANGES:
+        return;
+      case RESULT_APP_REVOKED:
+      case RESULT_NICKNAME_MISMATCH:
+        is_notify_system_ = true;
+        return;
+      case RESULT_CONSENT_NEEDED:
+      case RESULT_PERMISSIONS_REVOKED_AND_CONSENT_NEEDED: {
+        is_consent_needed_ = true;
+        break;
+        case RESULT_CONSENT_NOT_REQIURED:
```

Comment:
@AKalinich-Luxoft Fix indents

Reply:
@AByzhynar this code was removed, please review 89ceac249b54ec02c042865b27f4af8b2363d450